libs/design/card/src/card-base.directive.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Input,
+  Directive,
+  HostBinding,
+} from '@angular/core';
+
+import {
+  DaffArticleEncapsulatedDirective,
+  DaffColorableDirective,
+} from '@daffodil/design';
+
+/**
+ * This attribute determines what orientation the the card contents are stacked.
+ */
+export type DaffCardOrientation = 'vertical' | 'horizontal';
+export enum DaffCardOrientationEnum {
+  Vertical = 'vertical',
+  Horizontal = 'horizontal',
+}
+
+@Directive({
+  selector: '[daffCardBase]',
```

Comment:
Is it possible to remove this selector? I wouldn't want someone to accidentally use this directive.

Reply:
don't need to remove since this directive is not exposed in the public api.